resolwe/flow/views.py

```diff
@@ -118,6 +118,7 @@ def create(self, request, *args, **kwargs):
         ds_slug = request.data.get('descriptor_schema', None)
         if ds_slug:
             ds_query = DescriptorSchema.objects.filter(slug=ds_slug)
+            ds_query = get_objects_for_user(request.user, 'view_descriptorschema', ds_query)
             if not ds_query.exists():
```

Comment:
Why is this being handled with `.exists()` (also below)? I think it would be better to catch the `DoesNotExist` exception that `.latest()` will throw.